xfce/orage/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem 1.0
 
 name            orage
-version         4.12.1
+version         4.18.0
```

Comment:
It would now be customary to add a "revision 0" after the version. (In the deep past, we only put this in when the revision was 1 or above.)

Reply:
Fixed.